tests/test_secret.py

```diff
@@ -0,0 +1,73 @@
+import secrets
+
+import pytest
+from pydantic import ValidationError, parse_obj_as
+
+from fastapi_login.secrets import AsymmetricSecret, Secret, SymmetricSecret
+
+from .conftest import generate_rsa_key, require_cryptography
+
+happypath_parametrize_argvalues = [
```

Comment:
Why exactly `"happypath"`?

Reply:
"happypath" in terms of valid test case, they should pass given tests.

```python
def add(a, b):
    return a + b

# so 1 + 2 == 3 is a happypath of function add
def test_add_happypath():
     assert add(1,2) == 3
```